Geometry/CMSCommonData/python/cmsExtendedGeometry2015XML_cfi.py

```diff
@@ -219,8 +219,8 @@
         'Geometry/HcalCommonData/data/hcalouteralgo.xml',
         'Geometry/HcalCommonData/data/hcalforwardalgo.xml',
         'Geometry/HcalCommonData/data/average/hcalforwardmaterial.xml',
-        'Geometry/HcalCommonData/data/Phase0/hcalSimNumbering.xml', 
-        'Geometry/HcalCommonData/data/Phase0/hcalRecNumbering.xml', 
```

Comment:
@davidlange6 All run2 related files are kept in Geometry/HcalCommonData/data/Run2 while Geometry/HcalCommonData/data/Phase0 contains files relevant to Run1. Currently some of the files are identical but I am working on some changes which I shall submit later. The important thing for this PR is the material for 2016 period of Run2.
________________________________
From: David Lange [notifications@github.com]
Sent: 02 February 2016 13:43
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx64 Introduce changes needed for Hcal geometry for 2016 and 2017 (#13139)


In Geometry/CMSCommonData/python/cmsExtendedGeometry2015XML_cfi.py<https://github.com/cms-sw/cmssw/pull/13139#discussion_r51562931>:

> @@ -219,8 +219,8 @@
>          'Geometry/HcalCommonData/data/hcalouteralgo.xml',
>          'Geometry/HcalCommonData/data/hcalforwardalgo.xml',
>          'Geometry/HcalCommonData/data/average/hcalforwardmaterial.xml',
> -        'Geometry/HcalCommonData/data/Phase0/hcalSimNumbering.xml',
> -        'Geometry/HcalCommonData/data/Phase0/hcalRecNumbering.xml',


@bsunanda<https://github.com/bsunanda> - is there some difference between these files?

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/13139/files#r51562931>.